contrib/git-sync/demo/config/service.yaml

```diff
@@ -1,8 +1,11 @@
-id: blog-service
+apiVersion: v1beta3
 kind: Service
-apiVersion: v1beta1
-port: 80
-containerPort: 80
-selector:
-  name: blog
-createExternalLoadBalancer: true
+metadata:
+  name: blog-service
+spec:
+  createExternalLoadBalancer: true
+  ports:
+  - port: 80
+    protocol: TCP
```

Comment:
Do you mean 80 is default for TCP? I think it's just a convention. I'm not sure if k8s is making it default.